frontend/src/components/Search.tsx

```diff
@@ -1,77 +1,92 @@
 import { FunctionComponent, useMemo, useState } from "react";
 import { useNavigate } from "react-router-dom";
-import { Autocomplete, ComboboxItem, OptionsFilter, Text } from "@mantine/core";
+import {
+  ComboboxItem,
+  Image,
+  OptionsFilter,
+  Select,
+  Text,
+} from "@mantine/core";
+import { ComboboxItemGroup } from "@mantine/core/lib/components/Combobox/Combobox.types";
 import { faSearch } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { chain, includes } from "lodash";
 import { useServerSearch } from "@/apis/hooks";
 import { useDebouncedValue } from "@/utilities";
 
 type SearchResultItem = {
   value: string;
+  label: string;
   link: string;
+  poster: string;
+  type: string;
```

Comment:
Let's narrow down here:


```suggestion
  type: "show" | "movie";
```